docker-compose-base.yml

```diff
@@ -124,7 +124,7 @@ services:
       ADMIN_TOKEN: ${ODK_ADMIN_TOKEN}
 
       AETHER_KERNEL_TOKEN: ${KERNEL_ADMIN_TOKEN}
-      AETHER_KERNEL_URL: http://kernel:8000
+      AETHER_KERNEL_URL: http://kernel:8100
       AETHER_KERNEL_URL_TEST: http://kernel-test:9000
```

Comment:
Do you want to change the test ports? What about the minio container? :thinking:

Reply:
Yes, good points - I'll do the test ports, but I guess the minio container should go in another PR, since it's not just changing the ports. Could you do that one?

Reply:
yep!